### ISSUE TITLE
Deltas in reddit quotes

### ISSUE THREAD

**Issue by Snorrrlax** [MystK/delta-bot-three#185]
DeltaBot is written to ignore deltas included in reddit quotes. This is to allow a method of discussing deltas, or explaining how/when to award one, etc. 

However, sometimes users do attempt to award one in a reddit quote. I think this is due to reddit auto-filling a reply box with content you've copied, which is a method we encourage in the sidebar. 

I don't think we should change our system of ignoring reddit quotes, but I have another idea. 

The main problem here is if nobody replies to the user telling them that reddit quotes are ignored. Sometimes they do, or we receive a request in modmail, but sometimes they don't bother coming back. 

I think this would be avoided by an automatic PM (not a comment) from DeltaBot, like so:

> Hi, username.
>
> Your recent comment in /r/changemyview includes a delta in a reddit quote, which I'm programmed to ignore. This is to provide an easy way for users to explain how/when to use deltas, or discuss a comment that includes one, without accidentally awarding one themselves. 
> 
> **If you were trying to award a delta:** Please edit [your comment](www.reddit.com) to remove the '>' in front of it, and make sure you have explained how your view has changed. I will then automatically recheck your comment.
>
> If you understand that reddit quotes shouldn't be used when awarding deltas, please **reply to this PM with STOP** to prevent me from sending this message to you in the future. 
>
> Thanks!

If DeltaBot receives a STOP reply, then that username is added to a blacklist in the wiki.

Does this sound feasible?

**Comment by MystK** [comment]
Yes, this is possible :)

**Comment by kevinwern** [comment]
As mentioned on Slack, I'm starting work on this bug.

**Comment by Snorrrlax** [comment]
@kevinwern any luck with this so far?

**Comment by kevinwern** [comment]
Yeah, I've been working on it this week. I'll probably have something this weekend.

**Comment by Snorrrlax** [comment]
@kevinwern any update?

**Comment by kevinwern** [comment]
I thought I commented on this but I didn't...I did a bit of work but I've been swamped with school since then. I'll take myself off the issue. I should be able to work on it in December if someone else still hasn't picked it up.

So sorry again! I didn't mean to hog this bug and I hope other people get the chance to work on it.

**Comment by Snorrrlax** [comment]
Okay, no worries! 

Certainly - if it's still open in December, please feel free to pick it up again.

**Comment by Snorrrlax** [comment]
@kevinwern is there a possibility of you working on this again?
